runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -3035,32 +3046,40 @@ J9::Z::TreeEvaluator::multianewArrayEvaluator(TR::Node * node, TR::CodeGenerator
    generateRRInstruction(cg, TR::InstOpCode::LGR, node, temp2Reg, firstDimLenReg);
    generateRILInstruction(cg, TR::InstOpCode::MSGFI, node, temp2Reg, zeroArraySize);
 
+   /* temp2Reg =             temp2Reg             +                     temp1Reg
+    *          = [firstDimLenReg * zeroArraySize] + [array stride (bytes) + contiguousArrayHeaderSizeInBytes]
+    */
    generateRRInstruction(cg, TR::InstOpCode::AGR, node, temp2Reg, temp1Reg);
```

Comment:
Let's try not to mix comment syntax. First line is also self explanatory. How about:

```
cursor = generateRRInstruction(cg, TR::InstOpCode::AGR, node, temp2Reg, temp1Reg);
iComment("Calculates (firstDimLen * zeroArraySize) + (arrayStrideInBytes + arrayHeaderSize");
```